package.json

```diff
@@ -33,6 +33,7 @@
     "cf-nodejs-logging-support": "^6.5.1",
     "cfenv": "^1.2.3",
     "cors": "^2.8.5",
+    "country-code-lookup": "0.0.18",
```

Comment:
I've recently added i18n-iso-countries to the list of dependencies. It's a more mature code. Please use it instead.

Reply:
Okay, I now use i18n-iso-countries.
The question is: Is the country in the address always in English?

Reply:
No, but that npm package support all languages.

Reply:
Yes, at the moment the input parameter for the function -> 'en' is hard coded because I have no information about in which language the country name is.
That's why I asked if the address is always in English.

Reply:
Have a look on how I've used i18n-iso-countries with the country language.

Reply:
country: countries.getAlpha3Code(site.address.country, CountryLanguage.getCountryLanguages(options.countryID, (err, languages) => languages[0].iso639_1))

I am not sure if this working in all cases. If we are using the countryID, that means that the country name must be in the local language of the operator (in most cases not English)

German Operator DE*XYZ
countryID = 'DE' and country = 'Germany' would result in undefined for the Alpha3Code
countryID = 'DE' and country = 'Deutschland' would work.

But if there is a rule for the operators to insert the address in the local language and not in English, it will work.

Reply:
> country: countries.getAlpha3Code(site.address.country, CountryLanguage.getCountryLanguages(options.countryID, (err, languages) => languages[0].iso639_1))
> 
> I am not sure if this working in all cases. If we are using the countryID, that means that the country name must be in the local language of the operator (in most cases not English)
> 
> German Operator DE*XYZ
> countryID = 'DE' and country = 'Germany' would result in undefined for the Alpha3Code
> countryID = 'DE' and country = 'Deutschland' would work.
> 
> But if there is a rule for the operators to insert the address in the local language and not in English, it will work.

Google autocomplete places give us the country name in the country tongue. So it still can be an issue. You can add log when it's undefined.

Reply:
I added the error log in case the country code is undefined